src/main/java/com/nguyenquyhy/discordbridge/listeners/ClientConnectionListener.java

```diff
@@ -22,68 +24,74 @@
  */
 public class ClientConnectionListener {
     @Listener
-    public void onJoin(ClientConnectionEvent.Join event) {
-        DiscordBridge mod = DiscordBridge.getInstance();
-        GlobalConfig config = mod.getConfig();
+    public void onJoin(ClientConnectionEvent.Join event, @Root Player player) {
```

Comment:
@Root Player player has it filter the cause so a player is always present when this runs. It will then check for permission on that player.